Makefile

```diff
@@ -45,6 +45,13 @@ test-without-tsan: xcbeautify
 	   swift test --enable-code-coverage --disable-automatic-resolution \
 	2>&1 | ./xcbeautify
 
+test-query-performance: xcbeautify
+	set -o pipefail \
+	&& env RUN_QUERY_PERFORMANCE_TESTS=true \
+	   swift test --enable-code-coverage --disable-automatic-resolution \
+	   --filter QueryPerformanceTests \
+	2>&1 | ./xcbeautify
+
```

Comment:
If we set this up as a daily job we could get rid of this extra target entirely. It wouldn't matter if we _also_ run all the other tests. We'd just invoke

```
env RUN_QUERY_PERFORMANCE_TESTS=true make test-without-tsan
```